src/watchful.ts

```diff
@@ -86,15 +95,19 @@ export class Watchful extends Construct implements IWatchful {
       );
     }
 
-    this.dash = new cloudwatch.Dashboard(this, 'Dashboard', { dashboardName: props.dashboardName });
+    const dashboard = props.dashboard ?? true;
+    if (dashboard || props.dashboardName){
```

Comment:
Sorry, I confused myself a bit with the double negatives in the last review. This should actually throw an error if `dashboard` is `false` and `dashboardName` is provided
```suggestion
    if (props.dashboard === false && props.dashboardName) {
      throw new Error('Dashboard name is provided but dashboard creation is disabled');
    }
    if (props.dashboard !== false) {
```